### DISCUSSION TITLE
Same settings, different bar size and capture results.

### DISCUSSION THREAD

**Discussion by rcamarda390** [akumidv/tradingview-assistant-chrome-extension#199] [Ideas]
I already used my freetrial early in my strategy testing career, so I am unable to check:
I'd like the ability to test bar sizes for a given setting of an alert. I want to be able to step thru bar sizes, and capture metrics in a csv file. it would nice to have to capture errors as well, like the '9,000' deal exceeded, or when there is an overflow in the strategy.

**Comment by akumidv** [comment]
For saving alert settings there were extension on webstore, I've developed it by payments, so have agreement to not repeat if functionality. But it was a long time ago. So may be I'll implement something simple.

**Comment by akumidv** [comment]
About testing bar size alerts -  you can do this in pine script itself.

**Reply by rcamarda390** [reply]
True, if it were my pine script. I use subscription scripts which I don't have source code.
